pandas/plotting/_matplotlib/converter.py

```diff
@@ -891,16 +885,11 @@ class TimeSeries_DateLocator(Locator):
 
     Parameters
     ----------
-    freq : {var}
-        Valid frequency specifier.
-    minor_locator : {False, True}, optional
-        Whether the locator is for minor ticks (True) or not.
-    dynamic_mode : {True, False}, optional
-        Whether the locator should work in dynamic mode.
-    base : {int}, optional
-    quarter : {int}, optional
-    month : {int}, optional
-    day : {int}, optional
+    freq : {var} Valid frequency specifier. minor_locator : {False, True},
+        optional Whether the locator is for minor ticks (True) or not.
+        dynamic_mode : {True, False}, optional Whether the locator should work
+        in dynamic mode. base : {int}, optional quarter : {int}, optional month
+        : {int}, optional day : {int}, optional
```

Comment:
is this edit intentional?  looks like a few other docstrings in this module got mangled

Reply:
Thanks @jbrockmendel . I'd noticed that a few docstrings got autoformatted. This PR is just for tracking at the moment, but will sort that out at some point.